Library/Homebrew/cask/artifact/keyboard_layout.rb

```diff
@@ -0,0 +1,36 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cask/artifact/moved"
+
+module Cask
+  module Artifact
+    # Artifact corresponding to the `keyboard_layout` stanza.
+    #
+    # @api private
+    class KeyboardLayout < Moved
+      extend T::Sig
+
+      sig { returns(String) }
+      def self.english_name
+        "Keyboard Layout"
+      end
+
+      def install_phase(**options)
+        super(**options)
+        delete_keyboard_layout_cache(**options)
+      end
+
+      def uninstall_phase(**options)
+        super(**options)
+        delete_keyboard_layout_cache(**options)
+      end
```

Comment:
Is is always necessary to delete the layout cache on both install and uninstall? The PR linked to in the `homebrew/cask` repo only shows that this is done for one cask.

Reply:
I think so, https://www.kaufmann.no/roland/dvorak/macosx.html shows its removal in both installation and removal instructions.